File: Android/android101/E004/app/src/main/java/com/anas/learning/android/android101/e004/MainActivity.java
package com.anas.learning.android.android101.e004;

import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button showButton = findViewById(R.id.button);
        EditText editText = findViewById(R.id.editText);

        showButton.setOnClickListener(view -> {
            ((TextView) findViewById(R.id.textView))
                    .setText(editText.getText());
            Toast.makeText(super.getBaseContext(), "Hi", Toast.LENGTH_SHORT).show();
        });
    }
}

File: AndroidStudioProjects/FristApp/app/src/main/java/com/example/fristapp/MainActivity.kt
package com.example.fristapp

import android.content.Intent
import android.icu.util.Calendar
import android.os.Build
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.text.Editable
import androidx.annotation.RequiresApi
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    @RequiresApi(Build.VERSION_CODES.N)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var year = Calendar.getInstance().get(Calendar.YEAR)
        var v :Int = 2021 - he_age.text
            cal.setOnClickListener {

                println("$year")
                val intent=Intent(this,MainActivity2::class.java)
             intent.putExtra("age",he_age.text.toString())
             startActivity(intent)
         }
    }
}
